modules/remotecontrol/Receiver.js

```diff
@@ -268,6 +267,7 @@ export default class Receiver extends RemoteControlParticipant {
                         type: EVENT_TYPES.permissions,
                         action: PERMISSIONS_ACTIONS.grant
                     });
+                    this._sendStartAPIEvent();
```

Comment:
If the grant action and the api start event are always coupled, maybe they should be broken out into a function to encapsulate the behavior.